python/shotgun_model/shotgun_model.py

```diff
@@ -267,6 +269,16 @@ def get_entity_type(self):
         """
         return self.__entity_type
 
+    def get_additional_column_fields(self):
+        """
+        Returns the fields for additional columns and their associated column in the model.
+
+        :returns: A list of (column, field) tuples where column is the column number in the
+            model associated with the additional field.
+        """
+        # column is one greater than the index because of the default initial column
+        return [(i+1, field) for (i, field) in enumerate(self.__column_fields)]
```

Comment:
Sorry one more comment - maybe this should be returned as a dict keyed by index?
So that we could potentially return more data about columns?
I am thinking something along these lines:

```
1: {"field_name": "code", "type": "str", "display_name": "Shot Name"}
2: {"field_name": "entity", "type": "entity_link", "display_name": "Sequence"}
```

Please ignore if you feel I am over thinking things :)